app/src/components/InputChoiceGroup.tsx

```diff
@@ -7,7 +7,7 @@ interface Choice {
   checked: boolean
   handleChange: (e: React.ChangeEvent<HTMLInputElement>) => void
   labelKey: string
-  name?: string
+  name: string
```

Comment:
@karinamzalez I made this a required attribute so that the html `id` attributes could be unique. `id`s should be unique within any html page, so I combined name with value.